software/digitools.yml

```
@@ -0,0 +1,12 @@
+name: "Digitools"
+website_url: "https://ladigitale.dev/digitools"
+source_code_url: "https://codeberg.org/ladigitale/digitools"
+description: "A set of simple tools to accompany the animation of courses in person or from afar. (documentation in French)."
```

From afar reads a bit wierd. Remotely is a more standard term. 

```suggestion
description: "A set of simple tools to accompany the animation of courses in person or remotely. (documentation in French)."
```

---

Remotely was a weird term for me to figure out, but I'll roll with it 👍🏻